tests/unit/src/test/scala/tests/DefinitionSuite.scala

```diff
@@ -48,6 +48,7 @@ abstract class DefinitionSuiteBase(
         ScalaVersions.dialectForDependencyJar(jar.filename),
       )
     }
+    List(elems = 1)
```

Comment:
Why is this here?

Reply:
BecauseI was experimenting and totally forgot :sweat_smile: Thanks for noticing!

Reply:
Fixed